parley/src/layout/editor.rs

```diff
@@ -21,6 +21,21 @@ pub enum ActiveText<'a> {
     Selection(&'a str),
 }
 
+/// Opaque representation of a generation
+// Overflow handling: the generations are only compared,
+// so wrapping is usually fine, except that in the unlikely
+// event that exactly `usize::MAX` generations happen before the
+// application gets around to drawing `Generation(0)`.
+#[derive(PartialEq, Default, Clone, Copy)]
+pub struct Generation(usize);
```

Comment:
I think a `u32` is fine here.

Reply:
Agreed; I previously asked for clarification about the use of `usize` here.